Modules/Loadable/Markups/VTKWidgets/vtkSlicerROIRepresentation2D.cxx

```diff
@@ -190,13 +190,18 @@ void vtkSlicerROIRepresentation2D::UpdateCubeSourceFromMRML(vtkMRMLMarkupsROINod
     this->SetROISource(cubeSource);
     }
 
+  // Add a small amount to each axis so that the ROI is still visible when the slice view is parallel to
+  // and intersecting with one of the faces.
+  vtkMRMLSliceLogic* sliceLogic = this->GetApplicationLogic()->GetSliceLogic(this->GetSliceNode());
```

Comment:
These objects should be of course all valid, but just in case, who knows, it would be nice to add null-pointer checks (and we can use a default 1e-6 if we fail to get the actual thickness)

Reply:
Done.